src/align/struct_initializers.cpp

```diff
@@ -23,7 +23,7 @@ void align_struct_initializers()
 
       if (  prev->Is(CT_ASSIGN)
          && (  pc->Is(CT_BRACE_OPEN)
-            || (  language_is_set(LANG_D)
+            || (  language_is_set(lang_flag_e::LANG_D)
```

Comment:
Instead of the full `lang_flag_e::LANG_XXX` form, we could use `E_LANG_XXX` since they are define in `keywords.h`. It is quicker to read/write. Otherwise if we stick to the long form, it makes no much sense to have the  `E_LANG_XXX` one

Reply:
NO!
Using the short form produces:
```
src/align/struct_initializers.cpp:27:35: error: cannot convert ‘const size_t’ {aka ‘const long unsigned int’} to ‘lang_flag_e’
   27 |             || (  language_is_set(e_LANG_D)
      |                                   ^~~~~~~~
      |                                   |
      |                                   const size_t {aka const long unsigned int}
```